content/webapp/components/IIIFViewer/MainViewer.tsx

```diff
@@ -336,7 +336,7 @@ const ItemRenderer = memo(({ style, index, data }: ItemRendererProps) => {
           {displayItems.length > 0 &&
             displayItems.map(item => {
               return (
-                <ItemWrapper key={item.id}>
+                <ItemWrapper key={item.type + item.id}>
```

Comment:
Because sometimes `item` is a "Choice" array, the `id` can be `undefined` and that made it alert in the console as a non-unique key.